runtime/compiler/CMakeLists.txt

```diff
@@ -280,6 +280,11 @@ elseif(OMR_OS_AIX)
 	)
 	if(OMR_ENV_DATA64)
 		set(J9_SHAREDFLAGS -q64)
+		if(OMR_TOOLCONFIG STREQUAL "xlc")
+			# Modify the arch tuning value we inherit from omr
+			list(REMOVE_ITEM TR_COMPILE_OPTIONS "-qarch=pwr7")
+			list(APPEND TR_COMPILE_OPTIONS "-qarch=ppc64grsq")
```

Comment:
Does 32-bit (-qarch=ppc) need to be fixed as well?

Reply:
No. we get the correct value from OMR in that case
https://github.com/eclipse/omr/blob/0e780bb7e7d2c3c449ca8f2b2500b6121b8be00e/cmake/modules/platform/toolcfg/xlc.cmake#L43